lib/cli.rb

```diff
@@ -175,6 +175,7 @@ def burndown_init command = nil
   option 'with-fast-lane', :desc => "Plot Fast Lane with new cards bars", :required => false, :type => :boolean
   option 'no-tasks', :desc => "Do not plot tasks line", :required => false, :type => :boolean
   option 'push-to-api', :desc => 'Push collected data to api endpoint (in json)', :required => false
+  option 'board-id', :desc => "Id of Trello board", :required => false
```

Comment:
`:required => false` is the default, please remove it

and use new Ruby syntax and single quotes if there is not string interpolation:

``` Ruby
option 'board-id', desc: 'Id of Trello board'

Reply:
I disagree with the suddenly using single quotes when the existing codebase uses a different style. Except for `push-to-api`, every option uses double quotes for the description.

Reply:
We should change all of them. If you want you can include this in a different commit in this PR. But if not, we already have a Rubocop cop that will be enabled. But please do not introduce more offences.